webapp/src/main/java/com/box/l10n/mojito/monitoring/PrefixHierarchicalNameMapperStatsdConfigurationProperties.java

```diff
@@ -12,6 +12,7 @@
     String namePrefix = "";
     String tagKeyPrefix = "";
     NamingConvention namingConventionOverride = null;
+    String stripCharacters = null;
```

Comment:
i though having some javadoc here so that it shows when typing the properties in intellij for example but minor